docs/SPEC.md

```diff
@@ -3405,47 +3405,203 @@ If you need to convert other columns use the `map` function directly with the `u
+
+Example: `indexAny("v: chicken", chars: "aeiouy")` returns int `2`.
+
+##### isDigit
+
+Returns whether or not v is a digit.
+
+Example: `isDigit(v: "A")` returns boolean `false`.
+
+##### isLetter
+
+Returns whether or not v is a letter.
+
+Example: `isLetter(v: "A")` returns boolean `true`.
+
+##### isLower
+
+Returns whether or not v is lowercase.
+
+Example: `isLower(v: "A")` returns boolean `false`.
+
+##### isUpper
+
+Returns whether or not v is uppercase.
+
+Example: `isUpper(v: "A")` returns boolean `true`.
+
+##### joinStr
+
+Concatenates the elements of inputted string array to create a single string with v as the separator.
+
+Example: `joinStr(arr: []string{"a", "b", "c"}, v: ",")` returns string `a,b,c`.
+
+##### lastIndex
+
+Returns the index of the last instance of substr in s, or -1 if substr is not present in v.
+
+Example: `lastIndex(v: "go gopher", t: "go")` returns int `3`.
+
+##### lastIndexAny
+
+Returns the index of the last instance of any value from chars in v, or -1 if no value from chars is present in v.
+
+Example: `lastIndexAny(v: "go gopher", t: "go")` returns int `4`.
+
+##### repeat
+
+Returns a new string consisting of i copies of the string v.
+
+Example: `repeat("v: na", i: 2)` returns string `nana`.
+
+##### replace 
+
+Returns a copy of the string v with the first i non-overlapping instances of t replaced by u.
+
+Example: `replaceAll(v: "oink oink oink", t: "oink", u: "moo", i: 2)` returns string `moo moo oink`.
+
+##### replaceAll
+
+Returns a copy of the string v with the all non-overlapping instances of t replaced by u.
+
+Example: `replaceAll(v: "oink oink oink", t: "oink", u: "moo")` returns string `moo moo moo`.
+
+##### split
+
+Slices v into all substrings separated by t and returns a slice of the substrings between those separators.
+
+Example: `split(v: "a,b,c", t: ",")` returns []string `["a" "b" "c"]`.
+
+##### splitAfter
+
+Slices v into all substrings after each instance of t and returns a slice of the substrings between those separators.
+
+Example: `splitAfter(v: "a,b,c", t: ",")` returns []string `["a," "b," "c"]`.
+
+##### splitAfterN
+
+Slices v into all substrings after each instance of t and returns a slice of the substrings between those separators. i determines the number of substrings to return.
+
+Example: `splitAfterN(v: "a,b,c", t: ",", i: 2)` returns []string `["a," "b,c"]`.
+
+##### splitN
+
+Slices v into all substrings separated by t and returns a slice of the substrings between those separators. i determines the number of substrings to return.
+
+Example: `splitN(v: "a,b,c", t: ",", i: 2)` returns []string `["a" "b,c"]`.
 
 ##### title
 
 Convert a string to title case.
 
 Example: `title(v: "a flux of foxes")` returns the string `A Flux Of Foxes`.
 
+##### toLower
+
+Convert a string to lower case.
+
+Example: `toLower(v: "KOALA")` returns the string `koala`.
+
+##### toTitle
```

Comment:
This functions seems redundant. It seems like it does the same thing as toUpper. Is it different?

Reply:
Nevermind. Looked at the Go docs. It is different.